FinderKeeper/Assets/Scripts/PhysControllerFP.cs

```diff
@@ -1,153 +0,0 @@
-﻿using UnityEngine;
```

Comment:
Where was this script from?

Reply:
I dropped it there a long long time ago when we first created the prototype. I have kind of just forgotten all about it and left it there. Will remove it.